libclc/generic/lib/atomic/atom_cmpxchg.cl

```diff
@@ -6,7 +6,28 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include <clc/clc.h>
+#include <clc/atomic/atom_cmpxchg.h>
+#include <clc/atomic/atomic_cmpxchg.h>
+
+// cl_khr_global_int32_base_atomics
+#define IMPL(TYPE) \
```

Comment:
This doesn't look clang-formatted.

Reply:
done in https://github.com/llvm/llvm-project/pull/134489/commits/46d2df78452cf3e4518e847a60330d6e32ed2ad4

I'm also adding .cl file clang-format support in https://github.com/llvm/llvm-project/pull/134529

Reply:
Oh nice one, yeah. That explains why it isn't picked up upon!